workflow/rules/maftools.smk

```diff
@@ -0,0 +1,36 @@
+rule group_bcf_to_vcf:
+    input:
+        "results/final-calls/{group}.{event}.{calling_type}.fdr-controlled.bcf",
+    output:
+        temp("results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf"),
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    params:
+        extra="",
+    wrapper:
+        "v3.8.0/bio/bcftools/view"
+
+
+rule group_vcf_to_maf:
+    input:
+        vcf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf",
+        ref=genome,
+    output:
+        maf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.maf",
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    conda:
+        "../envs/vcf2maf.yaml"
+    params:
+        genome_build=config["ref"]["build"],
+        species=config["ref"]["species"],
```

Comment:
```suggestion
        genome_build=lookup(dpath="ref/build", within=config),
        species=lookup(dpath="ref/species", within=config),
```

Reply:
Commit is here: https://github.com/snakemake-workflows/dna-seq-varlociraptor/pull/297/commits/cce39c9b4a14118af5adef8647d996971fcfbe6b